README.md

```diff
@@ -35,7 +35,8 @@ Input argument | Meaning
 ---------|---------
 --restricted-licence (-r) | The name of the licence to restrict. Repeat this flag to specify multiple values.
 --ignore-project (-i) | Project which licence will not be checked for compliance. Repeat this flag to specify multiple values.
---override-licence (-o) | Can be used to override the licence detected for a project - e.g. github.com/spf13/cobra=MIT. Repeat this flag to specify multiple values.
+--override-licence (-o) | Can be used to override the licence detected for a project directory - e.g. vendor/github.com/spf13/cobra=MIT. Repeat this flag to specify multiple values.
```

Comment:
These flags are now mutually exclusive: `override-licence`, `override-licence-module`.
We should mention it and also also validation to reject invokation when both flags are used

Reply:
Can we have usage examples for both flags in the README? e.g. when using vendor, when using go modules